documentation/APIs/core-models.md

```diff
@@ -0,0 +1,576 @@
+# Core Models
+
+This document defines core models for IOTA V3 protocol.
+
+* [Block](#block)
+  * [Block ID](#block-id)
+  * [Block Syntactic Validation Rules](#block-syntactic-validation-rules)
+* [Slot Index](#slot-index)
+* [Slot Commitment](#slot-commitment)
+  * [Slot Commitment ID](#slot-commitment-id)
+* [Payloads](#payloads)
+  * [Tagged Data Payload](#tagged-data-payload)
+  * [Transaction](#transaction)
+  * [Outputs and Unlock blocks](#outputs-and-unlock-blocks)
```

Comment:
```suggestion
  * [Outputs and Unlocks](#outputs-and-unlocks)
```
Should `blocks` be removed? I think we did it in Stardust everywhere except for the actual blocks